docs/installation/configuration.md

```diff
@@ -94,11 +104,17 @@ By default a node connects to the Swarm mainnet.  When using a private or test n
 
 Any Bee node in a network can act as a bootnode.
 
-#### --config
+#### --clef-signer-enable             
 
-*default* `/home/<user>/.bee.yaml`
+*default* `false`
 
-The location of a yaml configuration file containing configuration instructions. See [configuration](#configuration-file).
+Set this to true to enable signing using Ethereum's 'Clef' external signer.
+
+#### --clef-signer-endpoint
```

Comment:
should be added somewhere that using clef only makes sense with a fitting rules files or if running in `--advanced` to allow for auto-signing of handshakes and cheques.